server/assets-v1/templates/src/pages/client_portal/login.html

```diff
@@ -166,27 +124,36 @@ <h1 class="font-bold text-4xl text-[#3751FE] mb-11 animate-slideFromLeft">Login<
           console.log("RESPONSE", responseTwo)
           if (responseTwo.token) {
             localStorage.setItem("clientToken", responseTwo.token);
-            alert("Login successful!");
+            showToastMessage("Login successful!", "success");
             window.location.href = "[[ .ProxyURL ]]/client-profile";
+          } else {
+            showToastMessage(responseTwo.message, "error");
           }
         } catch (error) {
```

Comment:
does this block executed if api giving 400 status code?

Reply:
Yes, exactly.

Reply:
No its not